taxonomy/models.py

```diff
@@ -640,6 +641,24 @@ class Meta:
 
         abstract = True
 
+    @classmethod
+    def get_whitelisted_job_skills(cls):
+        """
+        Get a QuerySet of whitelisted job skills.
+
+        White listed job skills are job skills with `is_blacklisted=False`.
+        """
+        return cls.objects.filter(is_blacklisted=False)
+
+    @classmethod
+    def get_blacklist_job_skill(cls):
+        """
+        Get a QuerySet of whitelisted job skills.
+
+        White listed job skills are job skills with `is_blacklisted=False`.
+        """
+        return cls.objects.filter(is_blacklisted=True)
+
```

Comment:
@saleem-latif Can we use a custom model manager in place of the above methods?

Reply:
I looked into that approach but it creates more problems. e.g. methods like `create_or_update`, `get_or_create` will not work correctly if we override the default model manager in our use case.